jenkins-scripts/docker/lib/boilerplate_prepare.sh

```diff
@@ -50,6 +50,27 @@ if [[ -z ${DO_NOT_CHECK_DOCKER_DISK_USAGE} ]]; then
           [[ -n $(sudo docker images -a -q) ]] && sudo docker rmi $(sudo docker images -a -q) || true
         fi
     fi
+
+    # if not enough, try to clean up build/ directories
+    PERCENT_DISK_USED=$(df -h ${docker_device} | grep ${docker_device} | sed 's:.* \([0-9]*\)%.*:\1:')
+    if [[ $PERCENT_DISK_USED -gt 85 ]]; then
+        MAX_SIZE_FOR_BUILD_DIRS="5G"
+        CMD_FIND_BIG_DIRS=(find ${HOME}/workspace -name build -exec du -h -d 0 -t ${MAX_SIZE_FOR_BUILD_DIRS} {} \;)
```

Comment:
is there a missing `$` after `=`? i.e. `=(` vs `=$(`

(`$` looks like a funny mustache)

Reply:
oh, maybe not; I am still a bash padawan

Reply:
Yeah this is setting the variable to a Bash array which is being used later on https://github.com/ignition-tooling/release-tools/pull/304/files#diff-0846c9d3ff0a72e925d87240401660f255512aa7a9eb532b39657209792ba5aeR62

Reply:
Quoting it makes it an array with a single element instead of each space-separated word its own.